pep/app/pods/most-viewed/template.hbs

```diff
@@ -9,12 +9,12 @@
                 }}
             </h4>
             <div class="d-flex align-items-center">
-                <Button
+                <Button::WithTooltip
                     @type="link"
                     @icon="download"
                     @iconSize="lg"
                     @action={{this.downloadCSV}}
-                    title={{t "common.export"}}
+                    @tooltip={{t "common.export"}}
```

Comment:
Same thing here, does this change accessibility?

Reply:
see comment on above instance.